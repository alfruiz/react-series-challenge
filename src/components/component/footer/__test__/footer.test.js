import React from 'react';
import { shallow } from 'enzyme';

import Footer from "./../index";

describe('<Footer> ', () => {

  let wraper;

  beforeAll(() => {
    wraper = shallow(
      <Footer />
    );
  });

  it('should be <footer></footer> type ', () => {
    expect(wraper.type()).toEqual('footer');
  });

  it('should be footer have a <div></div> children  ', () => {
    expect(wraper.find('div').type()).toEqual('div');
  });
});
