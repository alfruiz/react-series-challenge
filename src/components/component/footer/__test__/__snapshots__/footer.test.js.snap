// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer>  Will check with the snaptshot  1`] = `
<footer__StyledFooter
  className="footer"
>
  <div>
    Just a footer =)
  </div>
</footer__StyledFooter>
`;
